### DISCUSSION TITLE
Dark UI

### DISCUSSION THREAD

**Discussion by Sreesanth101** [Amazeryogo/Streamfish-Source-Code#6] [Ideas]
Any help on dark UI because colour codes in css are showing an error.

**Comment by Sreesanth101** [comment]
This is the current code
`body {
    background: color 212121;;
    color: 3B3B3B;
}`

**Comment by Amazeryogo** [comment]
yes I did add the code  212121 as bg and 3b3b3b as FG, just check

**Reply by Amazeryogo** [reply]
NOPE, it is not working at all, is the 3b3b3b even a colour?

**Reply by Sreesanth101** [reply]
it is but it doesnt work in .css files it only works in .html

**Reply by Sreesanth101** [reply]
a little late realisation 😅

**Comment by Sreesanth101** [comment]
hey, I found a way to make grey in css
you can you rgb(random numbers) and try or in vscode you can choose from the palate

**Reply by Sreesanth101** [reply]
try if `rgb(32, 34, 34)` works

**Reply by Amazeryogo** [reply]
ok, lemme see..

**Reply by Sreesanth101** [reply]
btw you can experiment it in google

**Reply by Sreesanth101** [reply]
by searching the rgb directly

**Reply by Sreesanth101** [reply]
![image](https://user-images.githubusercontent.com/72757090/105797510-20911e00-5fb6-11eb-968f-62f07fe7c35b.png)
